content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -1,29 +1,26 @@
 # Pipelines Files (`dvc.yaml`)
```

Comment:
It can become a project file now? It looks we can have plots now w/o any stages?

Reply:
what is the focus of this doc now, considering that we have the whole pipelines section?

Reply:
> It can become a project file now?

The term "Pipelines File" was already removed in another PR, it's just `dvc.yaml` now.

> what is the focus of this doc now

Yeah good question. It's becoming mostly a detailed schema reference for `dvc.yaml`

Reply:
Not for this PR: Maybe we need a new top-level **File Formats Reference** section before the Cmd Ref?

Reply:
> It's becoming mostly a detailed schema reference for dvc.yaml ...
Not for this PR: Maybe we need a new top-level Project File Formats (Reference) section

This PR has become pretty big already. So let's definitely work on the `dvc.yaml` ref. separately. I may just leave a few changes here that remove details from that doc (which have been moved to the new guide)...